### ISSUE TITLE
when set use_clearml = False，there is an error. 

### ISSUE THREAD

**Issue by EileenWang1228** [zetang94/ICSE2022_AST_Trans#2]
Dear Author:
when I set "use_clearml = False", there will be an error""ClearML configuration could not be found (missing `~/clearml.conf` or Environment CLEARML_API_HOST)\n"ValueError: ClearML configuration could not be found (missing `~/clearml.conf` or Environment CLEARML_API_HOST) To get started with ClearML: setup your own `clearml-server`, or create a free account at https://app.clear.ml". I guess the reason may be: in the exp_tracking.py, the variable "has_clearml"=True, however, I cannot set the variable to be False because there will be a RuntimeError "No experiment tracking system is setup. " "Please, setup either ClearML. ". What should I do to solve this problem.

when I set "use_clearml" = True, there will be an error "ValueError: ClearML configuration could not be found (missing `~/clearml.conf` or Environment CLEARML_API_HOST) To get started with ClearML: setup your own `clearml-server`, or create a free account at https://app.clear.ml" . I have no idea about the clearml. when I search on the Internet, I cannot find something helpful. I am looking for your help. Thank you.

**Comment by zetang94** [comment]
Yes, this is indeed a bug. For the moment, if you want to set use_clearml=False, I think it will be ok if line 15~22 in [exp_tracking.py](https://github.com/zetang94/ICSE2022_AST_Trans/blob/main/utils/exp_tracking.py) is commented. And if use_clearml is set to True, you need to init cleaml in your computer, just follow the init step [here](https://clear.ml/docs/latest/docs/getting_started/ds/ds_first_steps). 

I will fix this soon. Thank you :)
